### PR TITLE
Add WSL utils and handle setting clipboard text

### LINKED ISSUE #110: Copy to clipboard fails on WSL2
Attempting to copy to clipboard fails running under WSL2 just gives:

```
Failed to copy to clipboard
```

### PR DESCRIPTION
**Pull request by stuartleeks**
Fixes #110

### CODE REVIEW

**Comment by lawrencegripper** [comment]
Nice LGTM :)

File: internal/pkg/keybindings/globalhandlers.go [hunk 1]
```diff
@@ -48,7 +48,7 @@ func NewCopyHandler(content *views.ItemWidget, statusbar *views.StatusbarWidget)
 
 func (h CopyHandler) Fn() func(g *gocui.Gui, v *gocui.View) error {
 	return func(g *gocui.Gui, v *gocui.View) error {
-		var err error;
+		var err error
```
Comment (lawrencegripper) [inline internal/pkg/keybindings/globalhandlers.go]: :) Too much Java for you sir!

File: internal/pkg/wsl/utils.go [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+)
+
+// IsWSL attempts to determine if we're running on WSL
+func IsWSL() bool {
+	// If WSL_DISTRO_NAME env var is set then we're on WSL (unless someone is messing with us)
+	// Some early releases of WSL didn't set this IIRC
+	return os.Getenv("WSL_DISTRO_NAME") != ""
+}
+
+func TranslateToWindowsPath(localPath string) (string, error) {
```
Comment (lawrencegripper) [inline internal/pkg/wsl/utils.go]: Build wants a comment on this method then I think your good to go.

**Comment by lawrencegripper** [comment]
@stuartleeks I think your on holiday. No worries if you want to pick this up when your back!

File: internal/pkg/keybindings/globalhandlers.go (hunk 1 above)
Comment (stuartleeks) [reply internal/pkg/keybindings/globalhandlers.go]: hahaha!

**Comment by stuartleeks** [comment]
Just rebased as it said the branch needed updating. Will need re-approval @lawrencegripper  :-)

**Pull request merged**